README.md

```diff
@@ -152,7 +160,7 @@ The Python libraries you will be installing in the next step require the followi
 
 Once all the dependencies are in order, please do the following:
 
-    $ git clone git@github.com:dimagi/commcare-hq.git
+    $ git clone https://github.com/dimagi/commcare-hq.git
```

Comment:
any reason for this? just curious. i tend to prefer `ssh` clones over `https` for the passwordless pushes and pulls